files/dbsync/rpc/get_pool_owners.json

```diff
@@ -1,29 +1,31 @@
 {
-  "function": "get_pool_owners(_pool_tx_id)",
+  "function": "get_pool_owners(_pool_bech32)",
   "description": "Get registered pool owners",
   "parameters": [
     {
-      "name": "_pool_tx_id",
-      "description": "tx_id from get_pool_update() function"
+      "name": "_pool_bech32",
+      "description": "pool id in bech32 format"
     }
   ],
   "example": [
     {
       "type": "rest",
-      "command": "curl -d _pool_tx_id=142292 -s http://localhost:8050/rpc/get_pool_owners"
+      "command": "curl -d _pool_bech32=pool1z2ry6kxywgvdxv26g06mdywynvs7jj3uemnxv273mr5esukljsr -s http://localhost:8050/rpc/get_pool_owners"
     }
   ],
   "sql": [
-    "CREATE OR REPLACE FUNCTION get_pool_owners(_pool_tx_id numeric)",
+    "CREATE OR REPLACE FUNCTION get_pool_owners(_pool_bech32 text)",
     "RETURNS JSON STABLE LANGUAGE PLPGSQL AS $$",
     "BEGIN",
       "RETURN ( SELECT json_agg(js) json_final FROM ( SELECT json_build_object(",
-          "'owner_hash', encode(po.hash::bytea, 'hex'),",
+          "'owner_addr', tout.address,",
           "'reward_addr', sa.view",
         ") js",
         "FROM pool_owner AS po",
-        "LEFT JOIN stake_address AS sa ON RIGHT(encode(sa.hash_raw::bytea, 'hex'), -2) = encode(po.hash::bytea, 'hex')",
-        "WHERE po.registered_tx_id = _pool_tx_id",
+        "LEFT JOIN stake_address AS sa ON sa.id = po.addr_id",
```

Comment:
addr_id is not a column of pool_owner table. 
I think this has to be reverted to previous SQL command:
`LEFT JOIN stake_address AS sa ON RIGHT(encode(sa.hash_raw::bytea, 'hex'), -2) = encode(po.hash::bytea, 'hex')`

Reply:
I have that column:

![2021-06-09 23_01_07-pgAdmin 4](https://user-images.githubusercontent.com/12885604/121435378-ad8db000-c976-11eb-9e14-d31583efa428.png)

Reply:
Ok, updating dbsync on my end 👍